bouncer/shared/perform_swap.ts

```diff
@@ -362,17 +374,18 @@ export async function performVaultSwap(
   }[] = [],
 ): Promise<VaultSwapParams> {
   const tag = swapTag ?? '';
```

Comment:
also the swapTag should be the same, should be part of the logger that's passed in